index.yaml

```diff
@@ -33,13 +33,45 @@ entries:
     urls:
     - assets/ambassador/ambassador-6.7.1100+up6.7.11.tgz
     version: 6.7.1100+up6.7.11
+  - annotations:
```

Comment:
This was something we just discovered is happening in rancher/charts due to a bug in the build scripts. TLDR is that Helm does not update the index.yaml for new charts that match an existing chart's semver constraints (e.g. if it sees the +up annotation version, it ignores the chart version without that).

See https://github.com/rancher/charts/pull/1304#discussion_r658930831.